templates/bundles/DemosPlanCoreBundle/DemosPlanMap/map_admin_gislayer_edit.html.twig

```diff
@@ -258,6 +258,13 @@
                 </label>
             {% endif %}
             {% if hasPermission('feature_map_layer_visibility') %}
+                <dp-inline-notification
+                    id="defaultVisibilityHint"
+                    class="u-mb-0_5"
+                    message="{{ 'hint.gislayer.default.visibility'|trans }}"
+                    type="info"
+                    style="display: none;">
```

Comment:
We should avoid using style attributes, because they are normally difficult to maintain. We could use the tailwind `hidden` class instead, and then we should also adjust the code in `GisLayerEdit.js` to add/remove the class